DOCS/waf-buildsystem.rst

```diff
@@ -44,11 +44,11 @@ tailored to the project's specific needs.
 mpv's custom configure step on top of waf
 =========================================
 
-To some extents mpv has a custom build system written on top of waf. This
-document will not go over the standard waf behaviour as that is documented in
+To some extent mpv has a custom build system written on top of waf. This
+document will not go over the standard waf behavior as that is documented in
 the `Waf book <http://docs.waf.googlecode.com/git/book_17/single.html>`_.
 
-All of the configuration process is handled with a declarative approach. Lists
+All of the configuration processes are handled with a declarative approach. Lists
```

Comment:
Not sure how this plural is being justified. "The entirety of the configuration process is" may have been closer to the original intent here. Not that it really matters.

Reply:
"All the" makes no much sense in an individual noun

Reply:
"All the" makes no much sense in an individual noun